leasing/views.py

```diff
@@ -64,6 +64,15 @@ class InvoiceViewSet(viewsets.ModelViewSet):
     serializer_class = InvoiceSerializer
     filter_class = InvoiceFilter
 
+    @list_route(methods=['post'])
+    def create_invoices(self, request, pk=None):
+        """Runs the create_invoices management command"""
+        from django.core.management import call_command
+
+        call_command('create_invoices')
+
+        return Response()
```

Comment:
Ideally this could return the created invoices, but I guess this empty result is fine for the alpha.